hera_cal/frf.py

```diff
@@ -1289,7 +1293,7 @@ def tophat_frfilter_argparser(mode='clean'):
 def load_tophat_frfilter_and_write(datafile_list, case, baseline_list=None, calfile_list=None,
                                    Nbls_per_load=None, spw_range=None, cache_dir=None,
                                    read_cache=False, write_cache=False, external_flags=None,
-                                   factorize_flags=False, time_thresh=0.05,
+                                   factorize_flags=False, time_thresh=0.05, wgt_by_nsample=False, excluded_lsts=[],
```

Comment:
If the `excluded_lsts` parameter is updated anywhere in the function, then I think it's better to set the default to `None` and have something like `excluded_lsts = excluded_lsts or []` in the code. (Unless the optional parameter is a cache, default values should be immutable objects. You won't have any issues if the code doesn't modify the parameter, but it can lead to some interesting bugs if the code does modify it.)

Reply:
Done.